README.md

```diff
@@ -1,19 +1,152 @@
+
+You can download the zip from:
+#### latest [(master branch)](https://github.com/systems-cs-pub-ro/quiz-manager-moodle/zipball/master)
+
+Follow the steps below in your Moodle installation (administrative rights are required):
+
+* Go to `Site administration > Plugins > Install plugins`.
+* Upload the zip file. If you are prompted to add extra details, under `Plugin type` select `Question type (qtype)`.
+* Go to `Show more > Rename the root directory` and type `quizmanager`.
+* If your target directory is not writable, you will see a warning message. Proceed to [Manual installation](/README.md#manual-installation-from-zip-file).
+* Press `Install plugin from the ZIP file` button.
+
+### Manual Installation From zip file
+
+If you are prompted with the message `Plugin type location /your/moodle/path/question/type is not writable`, you have to copy the files manually to the server file system, using a zip file or [Git](/README.md#manual-installation-using-git).
+
+* Download the [zip file](https://github.com/systems-cs-pub-ro/quiz-manager-moodle/zipball/master).
+* Unzip the file in the correct location: `/your/moodle/path/question/type`.
+* Rename the directory to `quizmanager`.
+
+### Manual Installation Using Git
+
+To install using Git, use this command:
+
+```
+git clone https://github.com/systems-cs-pub-ro/quiz-manager-moodle.git /your/moodle/path/question/type/quizmanager
+```
+
+# Adding Questions
+
+Questions are added using the Moodle interface, after the plugin was installed, from the quiz menu.
+
+* After creating a quiz, access it and press the `Edit quiz` button.
+* On the right side of the page, go to `Add > + a new question` and select the `quizmanager` question type.
+* Select the category where you want your question to be from.
+* Set the difficulty from the given list _(Easy - Hard)_.
+* Set the topic that you want your question to have. Note: topics are case **insensitive**, and you can choose **only one** topic for one question.
+* Press the `Save changes` button.
+* Repeat the instructions for all the questions you want in your quiz.
+
+# Managing Questions
+
+The plugin can use any [question types](https://docs.moodle.org/311/en/Question_types) existent in the question bank, as long as the have the associated difficulty and topic tag.
+
+If questions are imported from an external source, using any [format](https://docs.moodle.org/311/en/Import_questions#Question_import_formats) supported by Moodle, the following rules must be followed:
+* The difficulty tag must be one from the following list: `Easy, Easy-Medium, Medium, Medium-Hard, Hard`.
+* The topic tag can take any form, as long as it is consistent with the set topic in the question edit form (see [Adding Questions](/README.md/#adding-questions)).
+* The `last_used` tag is optional (if not present will be considered 0), and the format is: `last_used:time`, where time is an integer representing time since the Epoch (00:00:00 UTC, January 1, 1970), measured in seconds.
```

Comment:
```suggestion
* The `last_used` tag is optional (if not present, it will be considered `0`).
  The format is: `last_used:time`, where time is an integer representing time since the Epoch (00:00:00 UTC, January 1, 1970), measured in seconds.
```